ai-platform-unified/notebooks/official/automl/automl-text-classification.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<blockquote>batch_prediction_job_name = "projects/1025771077852/locations/us-central1/batchPredictionJobs/6351346510034108416"</blockquote><p>Probably want to remove your project ID here and mark this as a placeholder (something that needs to be replaced). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
Nice catch!

Reply:
Done